.cirrus.yml

```diff
@@ -2,4 +2,10 @@ task:
   name: Test
   container:
     image: ghcr.io/cirruslabs/cirrus-cli:latest
-  script: cirrus internal test
+  test_script: cirrus internal test
+  setup_buildifier_script: |
+    apk add go
+    go install github.com/bazelbuild/buildtools/buildifier@latest
```

Comment:
@edigaryev I wonder if we should just include it into Cirrus CLI and add commands like `cirrus internal format` and `cirrus internal format --check` to check in a standard way. What do you think?

Reply:
Or `cirrus internal lint` instead of `cirrus internal format --check`

Reply:
I propose to keep it this way for now, until there's a wider demand for such a command in the Cirrus CLI.

The [`buildifier.go` command](https://github.com/bazelbuild/buildtools/blob/master/buildifier/buildifier.go) implements quite a lot of logic (on top of the [publicly exposed package](https://pkg.go.dev/github.com/bazelbuild/buildtools/buildifier)) which we'd need to copy/re-implement in the Cirrus CLI.